<ui version="4.0" >
 <class>wizDeviceFirst</class>
 <widget class="QWidget" name="wizDeviceFirst" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>271</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="textLabel1" >
     <property name="text" >
      <string>Mobile Phone Name:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KLineEdit" native="1" name="kcfg_devicename" />
   </item>
   <item>
    <widget class="QCheckBox" name="kcfg_autoload" >
     <property name="text" >
      <string>Loa&amp;d at startup</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="textLabel1_3" >
     <property name="text" >
      <string>Engine to use:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KComboBox" native="1" name="combo_engine" />
   </item>
   <item>
    <widget class="QLineEdit" name="kcfg_engine" />
   </item>
   <item>
    <widget class="KLineEdit" native="1" name="kcfg_mobileimei" />
   </item>
   <item>
    <widget class="QLabel" name="textLabel1_2" >
     <property name="text" >
      <string>Phone imei to search when probing devices:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>574</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QWidget</extends>
   <header>kcombobox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QWidget</extends>
   <header>klineedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_mobileimei</tabstop>
  <tabstop>kcfg_devicename</tabstop>
  <tabstop>kcfg_autoload</tabstop>
 </tabstops>
 <includes>
  <include location="local" >klineedit.h</include>
  <include location="local" >kcombobox.h</include>
  <include location="local" >klineedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
